source/user-manual/capabilities/vulnerability-detection/running_vu_scan.rst

```diff
@@ -86,47 +92,86 @@ The following fields are included in every alert:
 See below for examples of alerts:
 
 .. code-block:: none
-    :emphasize-lines: 5,10
+    :emphasize-lines: 5,26
     :class: output
 
-    ** Alert 1571137967.2083: - vulnerability-detector,gdpr_IV_35.7.d,
-    2019 Oct 15 11:12:47 c31dd66f7e82->vulnerability-detector
-    Rule: 23503 (level 5) -> 'CVE-2018-5710 on Ubuntu 18.04 LTS (bionic) - low.'
-    {"vulnerability":{"cve":"CVE-2018-5710","title":"CVE-2018-5710 on Ubuntu 18.04 LTS (bionic) - low.","severity":"Low","published":"2018-01-16T09:29:00Z","state":"Fixed","package":{"name":"libgssapi-krb5-2","version":"1.16-2ubuntu0.1","architecture":"amd64"},"condition":"Package less than 1.16.1-1","reference":"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-5710"}}
-    vulnerability.cve: CVE-2018-5710
-    vulnerability.title: CVE-2018-5710 on Ubuntu 18.04 LTS (bionic) - low.
-    vulnerability.severity: Low
-    vulnerability.published: 2018-01-16T09:29:00Z
-    vulnerability.state: Fixed
-    vulnerability.package.name: libgssapi-krb5-2
-    vulnerability.package.version: 1.16-2ubuntu0.1
+    ** Alert 1590407651.24259355: - vulnerability-detector,gdpr_IV_35.7.d,pci_dss_11.2.1,pci_dss_11.2.3,tsc_CC7.1,tsc_CC7.2,
```

Comment:
Coul you add an example of an alert from the Vendor feed and also one from both feeds?